frontend/src/metabase/lib/entity-id/hooks/use-validated-entity-id.ts

```diff
@@ -101,3 +119,46 @@ export const useValidatedEntityId = <
     return { id: null, isLoading: false, isError: true } as const;
   }, [isEntityId, isLoading, isError, entity_ids, id]);
 };
+
+export const fetchEntityId =
```

Comment:
I just realized we put this redux action inside a `hooks` folder. Which sounds really wrong.